docs/development.rst

```diff
@@ -380,76 +380,38 @@ The following examples show to extend the images.
 
   The images ship with Azul Zulu OpenJDK.  Due to licensing restrictions, we cannot bundle Oracle JDK, but we are testing on Zulu OpenJDK and do suggest it as a viable alternative.  In the event that you really need to use Oracle's version, you can follow the steps below to modify the images to include Oracle JDK instead of Zulu OpenJDK.
 
-  1. Change the base image to install Oracle JDK instead of Zulu OpenJDK.  First, you'll need to update the base image ``Dockerfile``:
+  1. Change the base image to install Oracle JDK instead of Zulu OpenJDK by updating ``debian/base/Dockerfile``.
```

Comment:
Clarified the location because it wasn't sufficiently clear IMHO for the average reader.